iphone/Classes/TiUIiPadPopoverProxy.m

```diff
@@ -259,47 +266,13 @@ - (void)cleanup
   [viewController.view removeObserver:self forKeyPath:@"safeAreaInsets"];
   RELEASE_TO_NIL(viewController);
   RELEASE_TO_NIL(popoverView);
-  [self performSelector:@selector(release) withObject:nil afterDelay:0.5];
+  [self release];
```

Comment:
The old statement was probably added ages ago, but I hope it's not causing further issues now.